### PR TITLE
Alert and button

### PR DESCRIPTION
**Pull request by veereshbv04**
Hi added Alert and Button components, please review and suggest improvements

Please check the outline button, is it possible to abstract more there ?

### CODE REVIEW

File: components/components_CSS/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+button{
+    border: none;
+}
+
+.btn{
+    
+    padding: 0.8rem 2.5rem;
+    color: white;
+    border-radius:  8px; ;
+    font-family: Georgia, 'Times New Roman', Times, serif;
```
Comment (nikhiljugale007) [inline components/components_CSS/Button/button.css:12]: you can define font in your root element instead or here.

File: components/components_CSS/Button/button.css [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    justify-content: center;
+    background-color: rgb(14, 177, 241);
+    border-radius: 2rem;
+    transition: transform 0.2s;
+    color: white;
+    cursor: pointer;
+}
+
+.float:hover{
+    height: 3.5rem;
```
Comment (nikhiljugale007) [inline components/components_CSS/Button/button.css]: use transform property instead on increasing font size.

**Comment by nikhiljugale007** [comment]
Suggestion on UI:
disabled button hover you can remove cursor by using curser: no-drop property
for the floating action button, I guess you are increasing font size which is shifting content instead use the transform property.

Added few comments in the code.

File: components/components_CSS/Button/button.css (hunk 2 above)
Comment (veereshbv04) [reply components/components_CSS/Button/button.css]: Thank you, changed to tranform now

**Comment by veereshbv04** [comment]
> Suggestion on UI: disabled button hover you can remove cursor by using curser: no-drop property for the floating action button, I guess you are increasing font size which is shifting content instead use the transform property.
> 
> Added few comments in the code.

I didnt noticed it, I fixed them now

File: components/components_CSS/Button/button.css (hunk 1 above)
Comment (veereshbv04) [reply components/components_CSS/Button/button.css:12]: I used different font inside button thats why i used font here, will try to abstract it

**Comment by Rishabh-078** [comment]
A couple of more suggestions:
1. In the alert, the text and dismiss "X" button are not aligned.
2.  In the icon button, background needs to be changed.

Rest looks good to me.

**Comment by nikhiljugale007** [comment]
> > Suggestion on UI: disabled button hover you can remove cursor by using curser: no-drop property for the floating action button, I guess you are increasing font size which is shifting content instead use the transform property.
> > Added few comments in the code.
> 
> I didnt noticed it, I fixed them now

Looking good now.

**Comment by veereshbv04** [comment]
> A couple of more suggestions:
> 
> 1. In the alert, the text and dismiss "X" button are not aligned.
> 2. In the icon button, background needs to be changed.
> 
> Rest looks good to me.

Thank you I will fix this

**Pull request merged**